matrimony_admin/views.py

```
@@ -1,16 +1,22 @@
 from django.contrib.auth import authenticate, login, logout
-from django.views.generic import FormView
+from django.views.generic import FormView, ListView
 from django.urls import reverse_lazy
-from .forms import AdminLoginForm, AdminProfileForm, NotificationDetailsForm
+from .forms import AdminLoginForm, AdminProfileForm,NotificationDetailsForm, EditUserForm
 from .models import BlockedUserInfo
 from U_auth.permissions import *
-
+from U_auth.models import UserPersonalDetails, Location
+from .forms import UserPersonalDetailsForm
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
```

Ruff

`django.core.paginator.Paginator` imported but unused

Remove unused import

(F401)